src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,166 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final String EXCEPTION_MESSAGE = "This index is out of bound ";
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
```

Comment:
what about renaming to head and tail? Optional, but helps to be consistent with LL impl